frontend/src/components/ApplicationSeason/reducer.tsx

```diff
@@ -22,6 +23,7 @@ export const initialState = {
     applicationPeriodEnd: minDate,
     applicationPeriodStart: minDate,
     end: minDate,
+    id: -1,
```

Comment:
Is this meant to initialize the ID to -1 if undefined? Can't you just make the currentSeason nullable?

Reply:
I'm changing it to 0, but the point is when the id is 0 it will not have inherited or fetched a season, meaning it is supposed to create a new one. Changing the layout and functionality

Reply:
Would it not make more sense to make currentSeason nullable if it doesn't exist, since it's a better reflection of what it actually is as @PeterRydberg suggests? And then handle undefined the same way as you to with ID 0 or -1 right name?

Reply:
I forgot which PR this was, my bad. This is resolved in #255